src/emulator/modules/SceDisplay/SceDisplay.cpp

```diff
@@ -77,53 +90,43 @@ EXPORT(int, sceDisplayUnregisterVblankStartCallback) {
 EXPORT(int, sceDisplayWaitSetFrameBuf) {
     STUBBED("move after setframebuf");
 
-    if (!host.display.sync_rendering) {
-        std::unique_lock<std::mutex> lock(host.display.mutex);
-        host.display.condvar.wait(lock);
-    } else
-        host.display.condvar.notify_all();
-
-    if (host.display.abort.load())
-        return SCE_DISPLAY_ERROR_NO_PIXEL_DATA;
-
-    return SCE_DISPLAY_ERROR_OK;
+    return display_wait(host);
 }
 
 EXPORT(int, sceDisplayWaitSetFrameBufCB) {
-    return UNIMPLEMENTED();
+    STUBBED("move after setframebuf");
+
+    return display_wait(host);
 }
 
 EXPORT(int, sceDisplayWaitSetFrameBufMulti) {
-    return UNIMPLEMENTED();
+    STUBBED("move after setframebuf");
+
+    return display_wait(host);
 }
 
 EXPORT(int, sceDisplayWaitSetFrameBufMultiCB) {
-    return UNIMPLEMENTED();
+    STUBBED("move after setframebuf");
+
```

Comment:
weird blank line

Reply:
is not waird, i have juust used same already set before for other unction, look just upside, all is samle :) and format said nothing

Reply:
Delete the blank line. Eyesore.

Reply:
hmm i have delete is very ugly wxithout space, i want keep like is on master space for separe stub info et return value
@VelocityRa if you can confirm for that ^^

Reply:
I don't mind the newline.

Reply:
yes, i have just wanted use same already present and used on master for this file( and other), without is to close and ugly to see